api/server/server.go

```diff
@@ -214,11 +214,21 @@ func postContainersPause(eng *engine.Engine, version version.Version, w http.Res
 	if err := parseForm(r); err != nil {
 		return err
 	}
-	job := eng.Job("pause", vars["name"])
-	if err := job.Run(); err != nil {
+
+	name := vars["name"]
```

Comment:
because I've added it in other server functions last time after the review.. I'm gonna create an issue to remove the check